images/ova/rhel-810.yaml

```diff
@@ -0,0 +1,25 @@
+---
+build_name: "rhel-810"
+packer_builder_type: "vsphere"
+guestinfo_datasource_slug: "https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo"
+guestinfo_datasource_ref: "v1.4.0"
+guestinfo_datasource_script: "{{guestinfo_datasource_slug}}/{{guestinfo_datasource_ref}}/install.sh"
```

Comment:
```suggestion

```
This is not needed anymore. We have to remove it from other OS's manifest files too. I will file separate PR.

Reply:
+1. Thanks for catching this, Shalin.

Reply:
lets handle it all in a new pr cleanly please!